bxsolana/provider/ws.ts

```diff
@@ -102,6 +102,20 @@ import {
 import { BaseProvider } from "./base"
 import { RpcWsConnection } from "../ws/rpcclient"
 import { RpcReturnType } from "../proto/runtime/rpc"
+import { Type as GetDriftMarketsRequest } from "../proto/messages/api/GetDriftMarketsRequest"
```

Comment:
these imports are weird, can you fix? you should be able to `import { GetDriftMarketsRequest } from "../proto/messages/api"`

Reply:
this is fixed everywhere now